build/stm32xx-sys/src/lib.rs

```diff
@@ -2,22 +2,24 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+use anyhow::Context;
 use quote::{quote, TokenStreamExt};
 use serde::Deserialize;
-use std::collections::BTreeMap;
+use std::{collections::BTreeMap, io::Write};
 
 #[derive(Deserialize, Default)]
-#[serde(rename_all = "kebab-case", deny_unknown_fields)]
```

Comment:
?

Reply:
Using `build_util::other_task_config` requires removing the `deny_unknown_fields`, because the current implementation of that function attempts to interpret _every_ task's configs as this type, which panics.